python.d/postgres.chart.py

```diff
@@ -217,91 +267,53 @@ def _connect(self):
         return True
 
     def check(self):
+        if not PSYCOPG2:
+            self.error('\'python-psycopg2\' module is needed to use postgres.chart.py')
+            return False
+        if not self._connect():
+            self.error('Can\'t connect to %s' % str(self.configuration))
```

Comment:
Looks like the original also had this defect. Might be an idea to just sanitize the configuration before printing as it's still useful information for debugging.